tools/cli_install.py

```diff
@@ -1,172 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Where will the CLI be installed during the test run? What version of the CLI is used?